src/components/GeneralQuestionManager/GeneralQuestionManager.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable react/style-prop-object */
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, ScrollView } from 'react-native';
+import { Button } from 'react-native-paper';
+import { getAllQuestionsOfType, getClient, setClient } from 'database/queries';
+import { Question, Client } from 'types/types';
+import LargeInput from 'components/LargeInput/largeInput';
+import SmallInput from 'components/SmallInput/smallInput';
+import Dropdown from 'components/Dropdown/dropdown';
+import Calendar from 'components/Calendar/calendar';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    height: '100%',
+    backgroundColor: '#fff',
+    display: 'flex',
+    paddingTop: 10,
+  },
+});
+
+export default function GeneralQuestionManager() {
```

Comment:
it'd be really helpful (for devs and posterity) to have a short blurb here documenting the function of this component!